src/features/user/TreeMapper/components/PlantLocationPage.tsx

```diff
@@ -186,11 +186,7 @@ export function LocationDetails({
             {t('coordinates')} <CopyToClipboard text={text} />
           </p>
           <div className={styles.value}>
-            <MaterialTextField
-              disabled={true}
-              variant="outlined"
-              value={text}
-            />
+            {text}
```

Comment:
If `deviceLocation` is null the UI shows `undefined`. It would be nicer to show some translation resource or just omit the display of coordinates at all:

<img width="1531" alt="Bildschirmfoto 2022-03-05 um 11 17 24" src="https://user-images.githubusercontent.com/1532418/156879459-5c4b5d2c-6b62-4588-be07-d13edd4f743d.png">

Reply:
Also the capture mode is not translated. I do not know which values except "external" will be possible.

Reply:
`external` is the only one(new).